zmon_worker_monitor/builtins/plugins/scalyr.py

```diff
@@ -49,6 +50,36 @@ def __init__(self, read_key, scalyr_region=None):
     def count(self, query, minutes=5):
         return self.timeseries(query, function='count', minutes=minutes, buckets=1, prio='low')
 
+    def logs(self, query, max_count=100, minutes=5, continuation_token=None):
+
+        if not query or not query.strip():
+            return {'messages': [], 'continuation_token': None}
+
+        val = {
+            'token': self.__read_key,
+            'queryType': 'log',
+            'maxCount': max_count,
+            'filter': query,
+            'startTime': str(minutes) + 'm',
+            'priority': 'low'
+        }
+
+        if continuation_token:
+            val['continuationToken'] = continuation_token
+
+        r = requests.post(self.__query_url, json=val, headers={'Content-Type': 'application/json'})
+
+        r.raise_for_status()
+
+        j = r.json()
+
+        if 'matches' in j:
```

Comment:
I think it would be better to handle errors based on Scalyr API: ``j['status'] == 'success'``
It seems that we don't handle this status consistently elsewhere, so we can have another round later for the other methods. ``logs`` will be the start :)